metricbeat/docs/fields.asciidoc

```diff
@@ -30378,13 +30378,6 @@ type: long
 *`elasticsearch.cluster.stats.nodes.data`*::
 +
 --
-type: long
-
---
-
-*`elasticsearch.cluster.stats.nodes.stats.data`*::
```

Comment:
Guessing https://github.com/elastic/elasticsearch/blob/main/x-pack/plugin/core/src/main/resources/monitoring-es-mb.json#L157-L162 should get removed as well?

Reply:
@matschaffer I think so? My old understanding was that those mappings were generated by the relevant `fields.yml`, files but I don't think that's the case anymore as PRs seem to manually remove things?

Reply:
I don't think they're autogenerated. It might be good to cross check with @klacabane but I recall we did a manual build of those templates based on fields.yml in the run up to 8.0 release. I don't think anyone ever automated the updates.

Reply:
Gotcha, I'll make a followup PR.

Reply:
PR: https://github.com/elastic/elasticsearch/pull/90649